website/docs/en/config/output.mdx

```diff
@@ -597,6 +597,10 @@ type ModulePathData = {
 };
 ```
 
+:::warning
+When debugging locally using [`@rspack/dev-server`](/config/dev-server) and [`HtmlRspackPlugin`](/plugins/rspack/html-rspack-plugin), the page entry and the chunks split from it by [`output.splitChunks`](/plugins/webpack/split-chunks-plugin) will be loaded by the HTML file with `script` tags. At this time, if the file name contains hash values, the hot update will fail due to the inability of dynamic loading of HTML.
```

Comment:
1. output.splitChunks -> optimization.splitChunks
2. Remove `@rspack/dev-server` because all dev servers have this problem (such as Rsbuild dev server).

```suggestion
When developing locally, it's recommended to avoid using hash values in filenames.

This is because entry files and chunks split by [`optimization.splitChunks`](/plugins/webpack/split-chunks-plugin) are loaded via `<script>` tags in HTML files. If filenames contain hash values, HMR will not work as HTML files cannot be dynamically loaded.
```